modules/tinymce/src/plugins/anchor/test/ts/browser/AnchorAlertTest.ts

```diff
@@ -0,0 +1,54 @@
+import { Pipeline, Step, Log, Waiter, UiFinder } from '@ephox/agar';
+import { TinyApis, TinyLoader, TinyUi } from '@ephox/mcagar';
+import AchorPlugin from 'tinymce/plugins/anchor/Plugin';
```

Comment:
Typo in `Anchor` here.

- [ ] Fix typo

Reply:
The typo came from the other test files so I'll fix those up as well.